### PR TITLE
SQS Remove warning and move to troubleshooting

### PR DESCRIPTION
**Pull request by danielmarbach**

### CODE REVIEW

**Comment by danielmarbach** [comment]
@dvdstelt I reworded it slightly. Please have another look

File: transports/sqs/troubleshooting.md [hunk 1]
```diff
@@ -54,3 +54,7 @@ Throttling errors are similar to any other technical error that can occur.
 As message sending does not happen within a handler context any failures during sending will not rely or be covered by the [recoverability feature](/nservicebus/recoverability/) mechanism. Any retry logic must be manually implemented.
 
 When throttling occurs with no custom error logic implemented, one or more messages might not have been transmitted to Amazon SQS. The custom retry logic could either retry all messages to be sent again, including already succeeded messages or only retry individual messages that failed.
+
+## On endpoint shutdown messages might be only visible after the visibility timeout has expired
+
+Due to an [issue in the AWS SDK](https://github.com/aws/aws-sdk-net/issues/796#issuecomment-375494537) after an endpoint shutdown, messages might indicate that they are in-flight for a longer time than expected. Messages will not be lost but may end up being visible in the input queue again up to 30 seconds later after an endpoint shutdown.
```
Comment (dvdstelt) [inline transports/sqs/troubleshooting.md]: I would add a comma

`Due to an [issue in the AWS SDK](https://github.com/aws/aws-sdk-net/issues/796#issuecomment-375494537),`

File: transports/sqs/troubleshooting.md [hunk 2]
```diff
@@ -54,3 +54,7 @@ Throttling errors are similar to any other technical error that can occur.
 As message sending does not happen within a handler context any failures during sending will not rely or be covered by the [recoverability feature](/nservicebus/recoverability/) mechanism. Any retry logic must be manually implemented.
 
 When throttling occurs with no custom error logic implemented, one or more messages might not have been transmitted to Amazon SQS. The custom retry logic could either retry all messages to be sent again, including already succeeded messages or only retry individual messages that failed.
+
+## On endpoint shutdown messages might be only visible after the visibility timeout has expired
+
+Due to an [issue in the AWS SDK](https://github.com/aws/aws-sdk-net/issues/796#issuecomment-375494537), after an endpoint shutdown, messages might indicate that they are in-flight for a longer time than expected. Messages will not be lost but may end up being visible in the input queue again up to 30 seconds later after an endpoint shutdown.
```
Comment (dvdstelt) [inline transports/sqs/troubleshooting.md]: It reads a little difficult. Maybe I don't get it either atm. But how about

> After an endpoint shutdown, messages might not be visible again until up to 30 seconds after the shutdown. The messages will not be lost but temporarily not available due to an issue in the AWS SDK.

File: transports/sqs/troubleshooting.md [hunk 3]
```diff
@@ -54,3 +54,7 @@ Throttling errors are similar to any other technical error that can occur.
 As message sending does not happen within a handler context any failures during sending will not rely or be covered by the [recoverability feature](/nservicebus/recoverability/) mechanism. Any retry logic must be manually implemented.
 
 When throttling occurs with no custom error logic implemented, one or more messages might not have been transmitted to Amazon SQS. The custom retry logic could either retry all messages to be sent again, including already succeeded messages or only retry individual messages that failed.
+
+## On endpoint shutdown messages might be only visible after the visibility timeout has expired
+
+After an endpoint is shutdown messages that were in flight but not handled might not be visible again in the input queue for up to 30 seconds (default message visibility time). Messages will not be lost and reappear in the input queue once the visibility timeout is elapsed. This is cause due to an [issue in the AWS SDK](https://github.com/aws/aws-sdk-net/issues/796#issuecomment-375494537).
```
Comment (mauroservienti) [inline transports/sqs/troubleshooting.md]:
```suggestion
After an endpoint is shutdown messages that were in flight but not handled might not be visible in the input queue for up to 30 seconds (default message visibility timeout). Messages will not be lost and will reappear in the input queue once the visibility timeout is expired. This is due to an [issue in the AWS SDK](https://github.com/aws/aws-sdk-net/issues/796#issuecomment-375494537).
```

Minor rewording.

**Pull request merged**